spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -346,6 +346,44 @@
           inline_error = strip_tags(t('doc_auth.errors.alerts.selfie_not_live_poor_quality'))
           expect(page).to have_content(inline_error)
         end
+        # New test
```

Comment:
I'm not sure this comment was meant to stay

Reply:
[Removed](https://github.com/18F/identity-idp/pull/10042/commits/15702df4b819b73f134e4d7f091276701d9ed99c).